File: cfgs/java_cfgs_foo/MINIMUM_XOR_VALUE_PAIR_1.java
Nodes
1;special;;;
2;statement;fun_1(foo_1);14;
3;statement;int minXor = Integer.MAX_VALUE;15;
4;statement;int val = 0;16;
5;statement;int i = 0;17;
6;conditional;foo_1 < (foo_2 - 1);18;
7;statement;val = foo_1[foo_2] ^ foo_3[foo_4 + 1];20;
8;statement;minXor = fun_1(foo_1, foo_2);21;
9;statement;i++;19;
10;statement;return foo_1;23;
11;exit;;;
Edges
0;2;
1;3;
2;4;
3;5;
4;6;
5;7;10;
6;8;
7;9;
8;6;
9;11;
10;

File: cfgs/pythoncfgs/MINIMUM_XOR_VALUE_PAIR_1.py
Nodes
1;statement;fun_1();10;
2;statement;foo_1 = fun_1(foo_2.float_info.max);11;
3;statement;foo_1 = 0;12;
4;conditional;foo < foo;13;
5;statement;return foo_1;16;
6;statement;foo_1 = foo_2[foo_3] ^ foo_4[foo_5 + 1];14;
7;statement;foo_1 = fun_1(foo_2, foo_3);15;
8;exit;;
Edges
0;2;
1;3;
2;4;
3;5;6;
4;8;
5;7;
6;4;
7;
